internal/aiptest/deletion/already_deleted.go

```diff
@@ -29,15 +29,18 @@ var alreadyDeleted = suite.Test{
 			f.P("created := fx.create(t)")
 		}
 		util.MethodDelete{
-			Resource: scope.Resource,
-			Method:   deleteMethod,
-			Name:     "created.Name",
-		}.Generate(f, "_", "err", ":=")
+			Resource:    scope.Resource,
+			Method:      deleteMethod,
+			ResourceVar: "created",
+		}.Generate(f, "deleted", "err", ":=")
 		f.P(ident.AssertNilError, "(t, err)")
+		// prevent no usage error
+		f.P("_ = deleted")
```

Comment:
we maybe generating some unnecessary code but its way simple than what I did [here](https://github.com/einride/protoc-gen-go-aip-test/pull/240/commits/ccaffdce07ae5333b069c833adcd61d7114cd156#diff-949601ae252056096b249929850f8a077a90d5f51fbbd5547dd6d28362b9a706R31-R39)

Reply:
hehe yeah all good!